### DISCUSSION TITLE
Input Volume is cut severely when Background Music is launched

### DISCUSSION THREAD

**Discussion by avirao123** [kyleneideck/BackgroundMusic#535] [Q&A]
For some reason, whenever I launch Background Music, my audio input is severely cut, and my friends are barely able to hear me when I am on a video call with them. Is there something I am doing wrong? Is there some way I can raise input volume through the app? Increasing the input volume in system preferences doesn't help. Thanks!

**Comment by kyleneideck** [comment]
It shouldn't affect your input devices at all. What app are you using for the video calls?

Have you tested it with any other apps, like Quicktime? I vaguely remember a similar problem with Facetime where Background Music was triggering Facetime's audio ducking feature.

**Reply by avirao123** [reply]
I am using FaceTime as a video call software. I become very quiet to my friends when I open background music. The audio ducking issue would make sense, but how do I solve that?

**Reply by kyleneideck** [reply]
Not sure. You could try disabling audio ducking as a workaround: https://discussions.apple.com/thread/251264880

But it seems strange that your input volume is being affected rather than your output volume. It'd be worth checking that you have the right input device selected in Facetime and that your input device's volume is set high enough. You can check that in `Audio MIDI Setup.app`.

Are you using Background Music to send your system audio over the Facetime call or something like that?

**Comment by antongeny** [comment]
I have the same issue. I made an Issues bug post about it. Only thing that resolves it is quitting the app. It affects both my output and input microphone. The person on the other end can barely hear me, I can barely hear them too. Quitting the app resolves it, and it crashes if I sift through Background Music menu options.
